node/src/components/block_synchronizer/tests.rs

```diff
@@ -201,7 +330,7 @@ async fn global_state_sync_wont_stall_with_bad_peers() {
     let event = mock_reactor.crank().await;
 
     let second_peer_set = peers.iter().copied().choose_multiple(&mut rng, 4);
-    check_sync_global_state_event(event, &block);
+    check_sync_global_state_event(event, block);
 
     // Wait for the latch to reset
     std::thread::sleep(Duration::from_secs(6));
```

Comment:
I think I'll request a tech-debt ticket to add an option to inject the latch reset time. Sleeping for 6 seconds in a test is such a waste of time... Wdyt?

Reply:
We have [this PR](https://github.com/casper-network/casper-node/pull/3810) which makes the latch configurable. Once that's merged we can change the latch interval here so that we don't waste time.